template.go

```diff
@@ -102,6 +112,22 @@ func groupByKeys(entries interface{}, key string) ([]string, error) {
 	return ret, nil
 }
 
+// groupByLabel is the same as groupBy but over a given label
+func groupByLabel(entries interface{}, label string) (map[string][]interface{}, error) {
+	getLabel := func(v interface{}) (interface{}, error) {
+		if container, ok := v.(RuntimeContainer); ok {
+			if container.Labels[label] == "" {
```

Comment:
Ah you are right, I misread `groupBy` as excluding the container if the value is blank, so I went this route for consistency, but it only excludes if the value is nil. I'll change it.